ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/Runner.java

```diff
@@ -130,23 +130,24 @@ private Router router() {
     router
         .route()
         .handler(
-            CorsHandler.create(buildCorsRegexFromConfig())
+            CorsHandler.create()
+                .addRelativeOrigin(buildCorsRegexFromConfig())
```

Comment:
Why do we need to make this change with addRelativeOrigin?

Reply:
The create(..) constructor has been deprecated. They have introduced addRelativeOrigin(s) methods instead.